docs/eventing/README.md

```diff
@@ -28,7 +28,7 @@ Knative Eventing is designed around the following goals:
 
 1. Knative Eventing resources are loosely coupled, and can be developed and deployed independently of each other.
 1. Event producers and event consumers are independent. Any producer can generate events before there are active event consumers that are listening. Any event consumer can express interest in an event or class of events, before there are producers that are creating those events.
-1. Knative Eventing is consistent with the [CloudEvents](https://github.com/cloudevents/spec/blob/master/spec.md#design-goals) specification that is developed by the [CNCF Serverless WG](https://lists.cncf.io/g/cncf-wg-serverless), which ensures cross-service interoperability.
+1. Knative Eventing is consistent with the [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/spec.md#design-goals) specification that is developed by the [CNCF Serverless WG](https://lists.cncf.io/g/cncf-wg-serverless), which ensures cross-service interoperability.
```

Comment:
It does make a big difference but I would point to the latest release, which is `v1.0.1`, so that people are aware it exists :-)

Reply:
@lionelvillard, thanks for pointing this out. I wasn't sure about this one so I've blindly followed other parts of documentation. I've added another commit that updates all links to CloudEvents docs master and v1.0 branches to v1.0.1 branch